services/QuillLMS/engines/evidence/app/views/evidence/research/gen_ai/application/_histogram.html.erb

```diff
@@ -0,0 +1,67 @@
+<style>
+  .histogram-wrapper {
+    display: flex;
+    align-items: flex-end;
+  }
+  .histogram-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+  .bars-container {
+    display: flex;
+    align-items: flex-end;
+    height: <%= @histogram_height %>;
```

Comment:
Seems like this height is hardcoded in the controller. You might just hardcode it here.